### ISSUE TITLE
[Request] Add to Scoop

### ISSUE THREAD

**Issue by EphemeralDev** [elijah-potter/harper#270]
It would be nice to be able to install/update via [Scoop](https://github.com/ScoopInstaller/Scoop) on windows. 

Scoop has the following requirements for its main bucket (cli apps).
- a reasonably well-known and widely used developer tool (e.g. if it's a GitHub project, it should have at least 500 stars and 150 forks)
- the latest stable version of the program
- the full version i.e. not a trial version
- a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)
- a non-GUI tool

As an example, here are the scoop manifests for [rust-analyzer](https://github.com/ScoopInstaller/Main/blob/master/bucket/rust-analyzer.json) and [marksman](https://github.com/ScoopInstaller/Main/blob/master/bucket/marksman.json)

**Comment by EphemeralDev** [comment]
I made a PR by using rust analyzer as a template, but looks like i need to fix the regex still. https://github.com/ScoopInstaller/Main/pull/6314

**Comment by elijah-potter** [comment]
Thank you so much for doing that. Let me know if there's anything I can do to help.

**Comment by EphemeralDev** [comment]
I think everything is good to go now, just need a maintainer to take a look at it. One question for you though, if you look at the [manifest](https://github.com/EphemeralDev/Main/blob/add-harper/bucket/harper.json) there is a field for the project homepage. Would you prefer the website or github be set there? I have it set to github currently.

**Comment by elijah-potter** [comment]
I think the information on GitHub is probably more relevant to the consumers of Scoop. 

Once your PR gets accepted, go ahead and close this issue. Thank you so much for doing this!

**Comment by EphemeralDev** [comment]
Merged. Installed and working in helix editor with a small edit of the languages config.
> Thank you so much for doing this!

No problem. Thanks for the opportunity to make my first contribution to a project.

**Comment by elijah-potter** [comment]
Of course! On that note, there is a [minimal page on how to set Harper up with Helix](https://writewithharper.com/docs/integrations/helix). Is there anything missing from there?

**Comment by EphemeralDev** [comment]
Nope that's pretty much all I used for harper specifically.  I also added it to my language server list for markdown so i could use it alongside marksman.

```
[[language]]
name = "markdown"
language-servers = ["marksman", "harper-ls"]
```
